src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -112,6 +124,11 @@ class CoreWorkerDirectTaskSubmitter {
   absl::flat_hash_map<WorkerAddress, std::shared_ptr<rpc::CoreWorkerClientInterface>>
       client_cache_ GUARDED_BY(mu_);
 
+  /// Map from worker address to the lease client through which it should be
+  /// returned.
+  absl::flat_hash_map<WorkerAddress, std::shared_ptr<WorkerLeaseInterface>>
```

Comment:
I found the naming of this map confusing. How about `worker_to_lease_client_`?

Reply:
Fixed.